README.rst

```diff
@@ -10,11 +10,16 @@ seaworthy
 
 Test harness for Docker container images
 
+Seaworthy has similar goals to `TestContainers`_, although it is written in
+Python and is based on our experience with testing Docker images.
```

Comment:
Based on the TestContainers docs, this isn't entirely true. While there's a lot of overlap, TestContainers seems to focus on providing test dependencies in containers whereas we focus specifically on testing that containers work.